pkg/cli/admin/release/mirror.go

```diff
@@ -140,13 +142,15 @@ func NewMirror(f kcmdutil.Factory, parentName string, streams genericclioptions.
 	}
 	flags := cmd.Flags()
 	o.SecurityOptions.Bind(flags)
+	o.ImageSourceOptions.Bind(flags)
 	o.ParallelOptions.Bind(flags)
 
 	flags.StringVar(&o.From, "from", o.From, "Image containing the release payload.")
 	flags.StringVar(&o.To, "to", o.To, "An image repository to push to.")
 	flags.StringVar(&o.ToImageStream, "to-image-stream", o.ToImageStream, "An image stream to tag images into.")
 	flags.StringVar(&o.FromDir, "from-dir", o.FromDir, "A directory to import images from.")
 	flags.StringVar(&o.ToDir, "to-dir", o.ToDir, "A directory to export images to.")
+	flags.StringVar(&o.ICSPTo, "icsp-to", o.ICSPTo, "Path to write ImageContentSourcePolicy file to.  If not set, will write to current directory.")
```

Comment:
I think it's about using the same mechanics for saving ISCP as in your proposal for saving release signatures.

Reply:
oh, the ICSP file can be saved to the "config" directory, in absence of a --icsp-to-dir flag - same as what we do with the signature file.  i'll make that change.

Reply:
I was going to do that initially, but the flag is `--release-image-signature-to-dir`  - should that be more generic for both files - the signature + icsp files? @wking

Reply:
Pushed a commit w/ a single flag `--release-config-dir` that sets the directory for both release-image-signature-dir and icsp-to dir.

Reply:
Has @smarterclayton signed off on the generic approach whose enhancement he rejected earlier (linked above)?  Because if that's back on the table, the generic option that it died with was [`--config-to-dir`][1].

[1]: https://github.com/openshift/enhancements/pull/188/files#diff-13e449411659f84a0f8827c8a909c727R131

Reply:
I rejected recursive handling, the idea of the command outputting multiple config to the same dir is fine.  If we already pick a namespace for ICSP, and we expect that you have to apply ICSP, why do we need more than one dir?

Reply:
I'd like the one directory, `--release-image-config-dir` that a user can provide for path to write either icsp-file and/or release-signature file.  So the current `-release-image-signature-to-dir` will go away in favor of `--release-image-config-dir`.    
(the --icsp-to flag above this comment is outdated, i've removed that and this PR has the 1 dir --release-image-config-dir)

Reply:
I've updated so that both flags will be accepted - deprecated message with `release-image-signature-to-dir` in favor of `release-image-config-dir`

Reply:
updated again - 2 flags - --release-image-signature-to-dir, --release-image-icsp-to-dir.  Both default to config or --to-dir dir/config